hardly/handlers/distgit.py

```diff
@@ -61,26 +118,40 @@ def run(self) -> TaskResults:
                 "Not creating a dist-git MR from "
                 f"{self.target_repo}:{self.target_repo_branch}"
             )
-            return TaskResults(success=True, details={})
+            return TaskResults(success=True)
 
         if not self.package_config:
             logger.debug("No package config found.")
-            return TaskResults(success=True, details={})
+            return TaskResults(success=True)
 
-        logger.debug(f"About to create a dist-git MR from source-git MR {self.mr_url}")
+        if (
+            self.target_repo_branch
+            not in self.packit.dg.local_project.git_project.get_branches()
+        ):
+            msg = f"Downstream {self.target_repo}:{self.target_repo_branch} branch does not exist."
+            self.project.get_pr(int(self.mr_identifier)).comment(msg)
+            logger.info(msg)
+            return TaskResults(success=True)
```

Comment:
We should probably put more text in the comment to guide contributors/maintainers.

Reply:
This was added in #56 and I don't think it'll happen too often because the source-git branches should in general match dist-git branches. But I tried to add more context to the message. Check the latest commit.